lib/views/doujin/doujin_library/doujin_card.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_rating_bar/flutter_rating_bar.dart';
+
+class DoujinCard extends StatefulWidget {
+  DoujinCard(
+      {Key? key,
+      required this.cover,
+      required this.mediaId,
+      required this.title,
+      required this.rating,
+      required this.id})
+      : super(key: key);
+
+  final String cover;
+  final String title;
+  final int rating;
+  final String mediaId;
+  final int id;
+
+  @override
+  State<DoujinCard> createState() => _DoujinCards();
+}
+
+class _DoujinCards extends State<DoujinCard> {
```

Comment:
![ref](https://cdn.discordapp.com/attachments/947386394156859422/953197304612003880/Group_21.png)

Style the cards according to the above reference, thanks.

Reply:
Maximum of 2 lines should be shown for the title, if it overflows, then ellipsis it.

Reply:
Don't worry about the font, will add that in the future.